README.md

```diff
@@ -79,9 +126,14 @@ more details on bouncycastle's cryptography features.
 
 </details>
 
+<br/>
+
+# License
```

Comment:
Do all of these sections really deserve to be `h1` headers? I don't think we need all these sections.